### ISSUE TITLE
Logs not showing up in logstash's own logfile until shutdown

### ISSUE THREAD

**Issue by jakommo** [jordansissel/ruby-cabin#38]
If only a few messages (2-3) are produced, they don't show up in logstash's own logfile until the process is shutdown.

Test with:
```
bin/logstash -e 'input { generator { count => 1000} } filter { ruby { code => "sleep 1" exclude_tags => [""] } } output { stdout { codec => rubydebug } }'  --log test.log
```
now run `tailf test.log`, it doesn't show anything.

If logstash is shutdown, the `deprecated` warning with the timestamp of startup suddenly appears.

**Comment by jordansissel** [comment]
This was fixed in a7c51dbf, but possibly not put into Logstash.

**Comment by jordansissel** [comment]
Published v0.7.2 just now with this fix.

**Comment by jordansissel** [comment]
The next release of Logstash should have this fixed. Sorry for the delay :)
